src/v2/guide/reactivity.md

```diff
@@ -1,83 +1,83 @@
 ---
-title: Reactivity in Depth
+title: La réactivité dans le détail
 type: guide
 order: 12
 ---
 
-<p class="tip">**Cette page est en cours de traduction française. Revenez une autre fois pour lire une traduction achevée ou [participez à la traduction française ici](https://github.com/vuejs-fr/vuejs.org).**</p>We've covered most of the basics - now it's time to take a deep dive! One of Vue's most distinct features is the unobtrusive reactivity system. Models are just plain JavaScript objects. When you modify them, the view updates. It makes state management very simple and intuitive, but it's also important to understand how it works to avoid some common gotchas. In this section, we are going to dig into some of the lower-level details of Vue's reactivity system.
+Nous avons couvert la plupart des principes fondamentaux, maintenant il est temps de plonger dans les méandres ! L'une des caractéristiques les plus évidentes de Vue est le système de réactivité discret. Les modèles sont simplement des objets JavaScript. Lorsque vous les modifiez, la vue est mise à jour. Cela rend le gestionnaire d'état très simple et intuitive, mais il est également important de comprendre comment cela fonctionne pour éviter quelques erreurs habituelles. Dans cette section, nous allons creuser certains détails de bas niveau du système de réactivité de Vue.
```

Comment:
dans les méandres... de quoi ? Il manque un bout à l'expression. Je suggère "dans les méandres de Vue" ou "plonger en profondeur"

One of Vue's most distinct features => L'une des fonctionnalités les plus caractéristiques/emblématiques de Vue

le système de réactivité non obstrusif

la vue se met à jour

Cela rend la gestion d'état

quelques erreurs classiques/communes

nous allons nous pencher sur certains détails